src/js/institutions.js

```diff
@@ -0,0 +1,140 @@
+module.exports = {
```

Comment:
Wouldn't hold up this PR for it, but we should probably move this, and others we'll be creating, into a subdirectory.